src/components/axes/numericAxis.ts

```diff
@@ -1,4 +1,4 @@
-//<reference path="../../reference.ts" />
+///<reference path="../../reference.ts" />
```

Comment:
This looked pretty serious. Is this caught by the linter as well? If not, it should